test/e2e/data/cbmt.mapml

```diff
@@ -0,0 +1,17 @@
+<mapml- xmlns="http://www.w3.org/1999/xhtml/">
+  <map-head>
+    <map-title>Canada Base Map - Transportation (CBMT)</map-title>
+    <map-meta http-equiv="Content-Type" content="text/mapml;projection=OSMTILE"></map-meta>
+    <map-meta charset="utf-8"></map-meta>
+    <map-link rel="license" href="https://www.nrcan.gc.ca/earth-sciences/geography/topographic-information/free-data-geogratis/licence/17285" map-title="Canada Base Map © Natural Resources Canada"></map-link>
```

Comment:
@prushforth I'm looking to reflect these changes in https://github.com/Maps4HTML/MapServer-documentation/tree/mapml, I'm wondering whether it was intended to change the `title` attribute on `<map-link>`(s) to `map-title` as well.

Reply:
I'm assuming that was not intended, as it seems no other attributes use the `map-` prefix.

Reply:
That's correct. We only changed the elements' prefix, I'll await further information as to whether it's necessary to change attributes. Fwiw I think the link title attribute would be appropriate if there's a conflict there that might be motivation to adopt map-title. We haven't done that yet, so I think it's best to keep with title for now.  Btw I can't find the mapserver MapML code, I was looking to update it too. Lete know if you come across it.

Reply:
MapServer doesn't actually support MapML yet, here's the implementation ticket: https://github.com/MapServer/MapServer/issues/6024.